test/framework/src/main/java/org/elasticsearch/test/rest/support/Features.java

```diff
@@ -34,7 +34,7 @@
  */
 public final class Features {
 
-    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key");
+    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key", "yaml");
```

Comment:
I think if you don't have the Java build stuff setup you should make javana
do the merge 😃

On Jul 14, 2016 8:21 PM, "Honza Král" <notifications@github.com> wrote:

> In
> test/framework/src/main/java/org/elasticsearch/test/rest/support/Features.java
> <https://github.com/elastic/elasticsearch/pull/19436#discussion_r70905675>
> :
>
> > @@ -34,7 +34,7 @@
> >   */
> >  public final class Features {
> >
> > -    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key");
> > +    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key", "yaml");
>
> Thank you @jasontedor <https://github.com/jasontedor>, I don't have my
> env setup for java so I missed this.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/19436/files/d53406b8d3919c5367c1bb574fd365fb2af7110e#r70905675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANLotcG0AVdYcNe3YtwU1U545rSEKT2ks5qVtJ2gaJpZM4JMfjQ>
> .
>